lib/sisimai/bite/email/einsundeins.rb

```diff
@@ -0,0 +1,143 @@
+module Sisimai::Bite::Email
+  module EinsUndEins
+    # Sisimai::Bite::Email::EinsUndEins parses a bounce email which created by
+    # 1&1. Methods in the module are called from only Sisimai::Message.
+    class << self
+      # Imported from p5-Sisimail/lib/Sisimai/Bite/Email/EinsUndEins.pm
+      require 'sisimai/bite/email'
+
+      Re0 = {
+        :from    => %r/\A["]Mail Delivery System["]/,
+        :subject => %r/\AMail delivery failed: returning message to sender\z/,
+      }
+      Re1 = {
+        :begin   => %r/\AThis message was created automatically by mail delivery software/,
+        :error   => %r/\AFor the following reason:/,
+        :rfc822  => %r/\A--- The header of the original message is following/,
+        :endof   => %r/\A__END_OF_EMAIL_MESSAGE__\z/,
+      }
+      ReFailure = {
+        mesgtoobig: %r/Mail[ ]size[ ]limit[ ]exceeded/x,
+      }
+      Indicators = Sisimai::Bite::Email.INDICATORS
+
+      def description; return '1&1: http://www.1and1.de'; end
+      def smtpagent;   return Sisimai::Bite.smtpagent(self); end
+      # X-UI-Out-Filterresults: unknown:0;
+      def headerlist;  return []; end
+      def pattern;     return Re0; end
+
```

Comment:
__[rubocop]__
* Style/Semicolon: Do not use semicolons to terminate expressions. [:ref](https://github.com/bbatsov/ruby-style-guide#no-semicolon)
* Layout/TrailingWhitespace: Trailing whitespace detected. [:ref](https://github.com/bbatsov/ruby-style-guide#no-trailing-whitespace)

<!-- Comment by [SideCI](https://sideci.com) -->